uvloop/handles/udp.pyx

```diff
@@ -188,3 +182,17 @@ cdef class UDPTransport(UVBaseTransport):
         # Ensure that what we buffer is immutable.
         self.buffer.append((bytes(data), addr))
         self._maybe_pause_protocol()
+
+    @functools.lru_cache()
+    def _validate_address(self, object addr):
```

Comment:
Can you try to make it a standalone function (not a method)?  That way the cache will be shared across different transports.

Reply:
Absolutely, done.